src/lapi.js

```diff
@@ -708,6 +708,11 @@ const lua_tocfunction = function(L, idx) {
 };
 
 const lua_tointeger = function(L, idx) {
+    let n = lvm.tointeger(index2addr(L, idx));
```

Comment:
Perhaps this should call `lua_tointegerx`?

Reply:
Perhaps, C Lua does ``#define lua_tointeger(L,i)      lua_tointegerx(L,(i),NULL)`` and ``
#define lua_tonumber(L,i)       lua_tonumberx(L,(i),NULL)``